stories/SQFormTextField.stories.tsx

```diff
@@ -45,7 +45,11 @@ const Template: SQFormTextFieldStoryType = (args) => {
       initialValues={{[defaultArgs.name]: ''}}
       {...SQFormProps}
     >
-      <SQFormTextFieldComponent {...rest} size={getSizeProp(size)} />
+      <SQFormTextFieldComponent
```

Comment:
Same as above.  I think adding a new story instead of changing the existing one might be preferable.

Reply:
@aharpt 
![image](https://user-images.githubusercontent.com/42131897/235917530-7b042d0b-8373-4e4e-b31e-2378bc8f0268.png)